cli/templatepresets_test.go

```diff
@@ -208,7 +213,69 @@ func TestTemplatePresets(t *testing.T) {
 		pty.ExpectRegexMatch(`preset-default\s+k1=v2\s+true\s+0`)
 		// The parameter order is not guaranteed in the output, so we match both possible orders
 		pty.ExpectRegexMatch(`preset-multiple-params\s+(k1=v1,k2=v2)|(k2=v2,k1=v1)\s+false\s+-`)
-		pty.ExpectRegexMatch(`preset-prebuilds\s+\s+false\s+2`)
+		pty.ExpectRegexMatch(`preset-prebuilds\s+Preset without parameters and 2 prebuild instances.\s+\s+false\s+2`)
+	})
+
+	t.Run("ListsPresetsJSON", func(t *testing.T) {
+		t.Parallel()
+
+		client := coderdtest.New(t, &coderdtest.Options{IncludeProvisionerDaemon: true})
+		owner := coderdtest.CreateFirstUser(t, client)
+		member, _ := coderdtest.CreateAnotherUser(t, client, owner.OrganizationID)
+
+		// Given: an active template version that includes presets
+		preset := proto.Preset{
+			Name:        "preset-default",
+			Description: "Preset wit parameters and 2 prebuild instances.",
```

Comment:
```suggestion
			Description: "Preset with parameters and 2 prebuild instances.",
```

Reply:
Nice catch 👀 Addressed in https://github.com/coder/coder/pull/19079/commits/2dea35554503048a2d691a3e8bb1d0490fb7b84b